cmd/server-main.go

```diff
@@ -277,7 +281,93 @@ func mergeServerCtxtFromConfigFile(configFile string, ctxt *serverCtxt) error {
 		ctxt.SFTP = append(ctxt.SFTP, fmt.Sprintf("ssh-private-key=%s", cf.Options.SFTP.SSHPrivateKey))
 	}
 
-	ctxt.Layout, err = buildDisksLayoutFromConfFile(cf.Pools)
+	ctxt.Layout, err = buildDisksLayoutFromConfFile(cf.EnableExpandPools, cf.Pools, false)
+
+	if ctxt.EnableExpandPools {
+		stat, err := os.Stat(configFile)
+		if err != nil {
+			return err
+		}
+		ltime := stat.ModTime()
+		go func() {
+			ticker := time.NewTicker(time.Second * 10)
+			for {
+				select {
+				case <-ticker.C:
+					lstat, err := os.Stat(configFile)
+					if err == nil {
```

Comment:
Again `if err != nil {  handle that }` so you don't get all this horrible indentation.